documentation/ci-fix.md

```diff
@@ -165,6 +165,13 @@ This validator is to ensure the cadl & swagger files in PR are consistent and th
 |InConsistentSwagger| the generated swagger is inconsistent with the swagger in PR, so you need to re-generate swagger from cadl, and check in it |
 |SwaggerNotExistInPR| the occurs when there is cadl file in the PR but the swagger is not present in the PR, so you need to add the swagger to the PR |
 
+## Traffic Validation
+
+It generates traffics for all operations defined in swaggers under default tag readme.md by using [RESTLer](https://github.com/microsoft/restler-fuzzer). Then it validates the request and response pairs from these traffics against correspondent swaggers. Finally, it provides a html report that reflects swagger accuracy.
```

Comment:
There are two parts here. One is restler coverage result, another is swagger accuracy report.  Do we cover how to fix issues by swaggerAccuracyReport?